src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/custom.py

```diff
@@ -1368,8 +1368,9 @@ def get_vmss_instance_view(resource_group_name, vm_scale_set_name, instance_id=N
     client = _compute_client_factory()
     if instance_id:
         if instance_id == '*':
-            return client.virtual_machine_scale_set_vms.list(resource_group_name, vm_scale_set_name,
-                                                             select='instanceView', expand='instanceView')
+            vm_list = list(client.virtual_machine_scale_set_vms.list(resource_group_name, vm_scale_set_name,
+                                                                     select='instanceView', expand='instanceView'))
```

Comment:
Actually, you don't have to convert the iterable to list here, because another list will be generated in the return statement anyway.

Reply:
I have to convert to list because I need to pull out the instance-view. That's why the return statement has the list comprehension.